ImageFeed/Helpers/AuthConfiguration.swift

```diff
@@ -0,0 +1,44 @@
+import Foundation
+
+enum Constants {
+    static let accessKey = "1UlRB2zD6MYqUW-nM042qpaYt3SPSRlIYAyvv_ik5fE"
+    static let secretKey = "Di64El2GK2aoz1mm0ihzExLXBOyhwWLz7KGuWZKzQIw"
+    static let redirectURI = "urn:ietf:wg:oauth:2.0:oob"
+    static let accessScope = "public+read_user+write_likes"
+    static let defaultBaseURL: URL = {
+        guard let url = URL(string: "https://api.unsplash.com") else {
+            preconditionFailure("Invalid Unsplash API base URL")
+        }
+        return url
+    }()
+    static let unsplashAuthorizeURLString = "https://unsplash.com/oauth/authorize"
+}
+
+struct AuthConfiguration {
+    let accessKey: String
+    let secretKey: String
+    let redirectURI: String
+    let accessScope: String
+    let defaultBaseURL: URL
+    let authURLString: String
+    
+    init(accessKey: String, secretKey: String, redirectURI: String, accessScope: String, authURLString: String, defaultBaseURL: URL) {
+        self.accessKey = accessKey
+        self.secretKey = secretKey
+        self.redirectURI = redirectURI
+        self.accessScope = accessScope
+        self.defaultBaseURL = defaultBaseURL
+        self.authURLString = authURLString
+    }
+    
+    static var standard: AuthConfiguration {
+        return AuthConfiguration(accessKey: Constants.accessKey,
+                                 secretKey: Constants.secretKey,
+                                 redirectURI: Constants.redirectURI,
+                                 accessScope: Constants.accessScope,
+                                 authURLString: Constants.unsplashAuthorizeURLString,
+                                 defaultBaseURL: Constants.defaultBaseURL)
+    }
```

Comment:
В данном случае для чтения файла лучше подходят такие отступы

```suggestion
        return AuthConfiguration(
            accessKey: Constants.accessKey,
            secretKey: Constants.secretKey,
            redirectURI: Constants.redirectURI,
            accessScope: Constants.accessScope,
            authURLString: Constants.unsplashAuthorizeURLString,
            defaultBaseURL: Constants.defaultBaseURL
        )
    }
```